openedx/core/lib/teams_config.py

```diff
@@ -1,17 +1,37 @@
 """
 Safe configuration wrapper for Course Teams feature.
 """
-
-
+import logging
 import re
 from enum import Enum
 
 from django.utils.functional import cached_property
+from django.utils.translation import gettext_lazy as _
+from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag
+
+from xmodule.partitions.partitions import UserPartition, UserPartitionError
+log = logging.getLogger(__name__)
 
 # "Arbitrarily large" but still limited
 MANAGED_TEAM_MAX_TEAM_SIZE = 200
 # Arbitrarily arbitrary
 DEFAULT_COURSE_RUN_MAX_TEAM_SIZE = 50
+TEAM_SCHEME = "team"
+TEAMS_NAMESPACE = "course_teams"
```

Comment:
@mariajgrimaldi I had forgotten that the flag `teams.enable_teams_app` existed! 

Could you change the new `TEAMS_NAMESPACE` so that our new flag is also under the same `teams.` namespace?

```suggestion
TEAMS_NAMESPACE = "teams"
```

Reply:
Thank you!